<?xml version="1.0"?>
<launch>

    <arg name="world_frame"          default="world" />
    <arg name="t265_world_frame"     default="realsense_t265_link" />
    <arg name="odom_frame"           default="odom" />
    <arg name="t265_odom_frame"      default="realsense_t265_link_" />
    <arg name="odom_topic"           default="/realsense_t265/odom/sample" />    

    <!--<node name="tf_t265_world_odom_glue" pkg="jetsoncar_driver" type="tf_t265_world_odom_glue.py" args="$(arg robot_frame) $(arg sensor_frame)" required="false" />-->
    <node name="tf_t265_world_odom_glue" pkg="jetsoncar_driver" type="tf_t265_world_odom_glue.py" required="false">
        <param name="world_frame"         type="str"   value="$(arg world_frame)" />
        <param name="t265_world_frame"    type="str"   value="$(arg t265_world_frame)" />
        <param name="odom_frame"          type="str"   value="$(arg odom_frame)" />
        <param name="t265_odom_frame"     type="str"   value="$(arg t265_odom_frame)" />
        <param name="odom_topic"          type="str"   value="$(arg odom_topic)" />
    </node>

</launch>
